src/components/dashboard/dashboard-view.js

```diff
@@ -4,42 +4,84 @@ import { bindActionCreators } from 'redux';
 import { translate } from 'react-i18next';
 import { translateOptions } from '../../i18n/config';
 import { Card } from 'semantic-ui-react';
+import { requestGetDashboardStats } from '../../actions/app-actions';
+import {MONITORING_STATUS_POLLING_DURATION} from '../../constants/misc'
```

Comment:
space after `{` and before `}` making it `{ MONITORING_STATUS_POLLING_DURATION }`